features/gezag-minderjarige/1.2-is-persoon-minderjarig-en-niet-overleden.feature

```diff
@@ -0,0 +1,111 @@
+  gezag (11.32.10). Zodra dit het geval is moet deze informatie worden meegenomen in de beoordeling en bij aanwezigheid van deze indicatie
+  wordt de persoon behandeld als meerderjarige.
+
+  Gebruikte velden:
+  - datum opschorting bijhouding (67.10)
+  - geboortedatum (03.10)
+
+  Achtergrond:
+    Gegeven de persoon 'Laura' met burgerservicenummer '000000012'
+    * is meerderjarig
+    En de persoon 'Ludo' met burgerservicenummer '000000024'
+    * is meerderjarig
+    En 'Laura' en 'Ludo' zijn met elkaar gehuwd
+    En de persoon 'Linda' met burgerservicenummer '000000036'
+    * is ingeschreven in de BRP
+    * is niet geëmigreerd geweest
+    * is in Nederland geboren
+    * heeft 'Laura' als ouder 1
+    * heeft 'Ludo' als ouder 2
+
+  Regel: gezag wordt niet bepaald wanneer het kind overleden is
+
+    Scenario: Wanneer gezag wordt opgevraagd van een overleden kind, wordt geen gezag teruggeven.
+      Gegeven persoon 'Linda'
+      * is minderjarig
+      En bijhouding van de persoonsgegevens van 'Linda' is opgeschort met de volgende gegevens
+      | datum opschorting bijhouding (67.10) |
+      | gisteren                             |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
+
+  Regel: gezag wordt niet bepaald wanneer het kind meerderjarig is
+
+    Scenario: Wanneer gezag wordt opgevraagd van een kind die meerderjarig is, wordt geen gezag teruggeven.
+      Gegeven persoon 'Linda'
+      * is meerderjarig
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
+
+  Regel: gezag wordt niet bepaald wanneer leeftijd onbekend is
+
+    Scenario: Wanneer gezag wordt opgevraagd van een kind waarvan leeftijd onbekend is, wordt geen gezag teruggeven.
+      Gegeven heeft de volgende gegevens
+      | geboortedatum (03.10) |
+      | 00000000              |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
+
+  Regel: het bepalen van gezag gaat verder wanneer het kind minderjarig is
+
+    Scenario: Wanneer gezag wordt opgevraagd van een kind die minderjarig is, wordt tweehoofdig ouderlijk gezag teruggeven.
```

Comment:
de andere features noemen het `is er sprake van TweehoofdigOuderlijkGezag`
Dat is geen conventie, maar misschien goed om consistent te zijn

Reply:
De scenario zinnen met "is er sprake van TweehoofdigOuderlijkGezag" zijn grammaticaal incorrect. Daarnaast vind ik het taalkundig onduidelijk. Alle recente voorgestelde scenario's wijken hier ook van af.